Monika After Story/game/script-stories.rpy

```diff
@@ -840,6 +840,42 @@ label mas_story_friend:
     m 1hua "I hope you enjoyed it, [player]!"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_story_database,
+            eventlabel="mas_story_tanabata",
+            prompt="The weaver girl and the cowherd",
+            unlocked=True
+
+        ),
+        code="STY"
+    )
+
+label mas_story_tanabata:
+    call mas_story_begin
+    m 6eub "[player], did you ever hear the story of the weaver girl and the cowherd?"
+    m 5sub "It's an old chinese story of two lovers."
+    m 1dub "Orihime, the daughter of the Jade Emperor, wove beatiful clothes by the bank of Amanogawa."

```

Comment:
I'd use some alternative translation instead of Jade Emperor here. Reasons being that:

- Most people don't have even passing familiarity with Chinese mythology. The name is ambiguous enough that it might be mistaken for an actual historical figure.
- Jade Emperor is mostly used in Chinese context. This creates inconsistency with the rest of the names here being Japanese.

Reply:
Hmm, the only other translation I could find was his title as ruler of heaven, and Tiān Gōng, which means heavenly grandfather. Should I maybe translate the sentence like this instead:

    m 1dub "Orihime, the daughter of the Jade Emperor, Ruler of Heaven, wove beautiful clothes by the bank of Amanogawa."